python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_check_selection.py

```diff
@@ -0,0 +1,126 @@
+import json
+from pathlib import Path
+
+from dagster import AssetCheckKey, AssetKey
+from dagster._core.definitions.asset_graph import AssetGraph
+from dagster_dbt import (
+    DagsterDbtTranslator,
+    DagsterDbtTranslatorSettings,
+    build_dbt_asset_selection,
+)
+from dagster_dbt.asset_decorator import dbt_assets
+
+dagster_dbt_translator_with_checks = DagsterDbtTranslator(
+    settings=DagsterDbtTranslatorSettings(enable_asset_checks=True)
+)
+
+manifest_path = (
+    Path(__file__)
+    .joinpath("..", "..", "dbt_projects", "test_dagster_asset_checks", "manifest.json")
+    .resolve()
+)
+
+with open(manifest_path, "r") as f:
+    manifest = json.load(f)
```

Comment:
This can be done in one line, since we're using `pathlib` :)
```suggestion
    manifest = json.load(manifest_path.read_bytes())
```